content/applications/sales/sales/amazon_connector/manage.rst

```diff
@@ -16,6 +16,13 @@ For *FBM* (Fulfilled by Merchant), the same is done for *Unshipped* and *Cancele
 synchronized order, a sales order and customer are created in Odoo (if the customer is not already
 registered in the database).
 
+.. important::
+   The stock synchronization does **not** currently support selling the same product in FBM *and*
+   FBA. At times, when stock is sent for all products, it triggers a stock problem with Amazon,
+   where Amazon incorrectly thinks the FBA product has some quantity in FBM. As a result, Amazon
+   then sells it as FBM, instead of taking from their own warehouse. Odoo developers are currently
+   working on resolving this issue to avoid future discrepancies.
```

Comment:
I like the formatting used in the other doc, and please add the abbr again :)
```suggestion
   The stock synchronization does **not** currently support selling the same product as :abbr:`FBM
   (Fulfilled By Merchant)` *and* :abbr:`FBA (Fulfilled By Amazon)`.

   At times, when stock is sent for all products, it triggers a stock problem with Amazon, where
   Amazon incorrectly thinks the :abbr:`FBM (Fulfilled By Merchant)` product has some quantity in
   :abbr:`FBM (Fulfilled By Merchant)`.

   As a result, Amazon then sells it as :abbr:`FBM (Fulfilled By Merchant)`, instead of taking from
   their own warehouse. Odoo developers are currently working on resolving this issue to avoid
   future discrepancies.
```